### PR TITLE
feat(benchmark): only mode

### PR DESCRIPTION
**Pull request by Aslemammad**
I'll address bench todo mode and reporters' issues in the next PRs.

### CODE REVIEW

File: test/benchmark/test/only.bench.ts [hunk 1]
```diff
@@ -0,0 +1,68 @@
+import assert from 'node:assert'
```
Comment (sheremet-va) [inline test/benchmark/test/only.bench.ts:1]: this file is not running by `test` script

and even then, it will never fail, because it just checks json

**Review by sheremet-va — CHANGES_REQUESTED** [review]

**Review by antfu — APPROVED** [review]

File: test/benchmark/test/only.bench.ts [hunk 2]
```diff
@@ -0,0 +1,59 @@
+import { bench, describe, expect } from 'vitest'
```
Comment (sheremet-va) [inline test/benchmark/test/only.bench.ts:1]: This file is still not running, I just put the previous comment on the first line. Please, ensure that your tests are passing before requiring review.

**Review by sheremet-va — CHANGES_REQUESTED** [review]

File: test/benchmark/test/only.bench.ts (hunk 2 above)
Comment (Aslemammad) [reply test/benchmark/test/only.bench.ts:1]: I see, I didn't know these benchmarks are not running, so misunderstood you! Let me see.

Comment (Aslemammad) [reply test/benchmark/test/only.bench.ts:1]: I see, I didn't know these benchmarks are not running, so misunderstood you! Let me see.

File: test/benchmark/package.json [hunk 3]
```diff
@@ -2,8 +2,6 @@
   "name": "@vitest/benchmark",
   "private": true,
   "scripts": {
-    "test": "node test.mjs",
-    "testu": "vitest -u",
-    "coverage": "vitest run --coverage"
+    "bench": "vitest bench"
```
Comment (sheremet-va) [inline test/benchmark/package.json:5]: Benchmarks never fail. I specifically wrote `test.mjs` to correctly test benchmarks output, and now two other files are not tested.

File: package.json [hunk 4]
```diff
@@ -19,6 +19,7 @@
     "lint": "eslint .",
     "lint:fix": "nr lint --fix",
     "release": "bumpp package.json packages/*/package.json --commit --push --tag && pnpm -r publish --access public",
+    "bench:all": "cross-env CI=true pnpm -r --stream run bench --allowOnly",
```
Comment (sheremet-va) [inline package.json:22]: We don't need this, we should only run tests in CI, it doesn't make sense to "just" run benchmarks in CI, we should test benchmark functionality, which is not testable with `vitest bench`.

**Review by sheremet-va — CHANGES_REQUESTED** [review]

File: test/benchmark/package.json (hunk 3 above)
Comment (sheremet-va) [reply test/benchmark/package.json:5]: Try to run a failing script first. Only after you ensured that tests __can actually__ fail CI, run an actual test.

Right now you are making _assumptions_ that something works, but reality disagrees.

Comment (Aslemammad) [reply test/benchmark/package.json:5]: Ok, Let me analyze test.js.

**Review by sheremet-va — APPROVED** [review]

**Review by sheremet-va — CHANGES_REQUESTED** [review]
I just noticed, there is no documentation for this feature. Please, add a short documentation alongside `bench.skip` (already in the docs)

**Review by sheremet-va — APPROVED** [review]

**Pull request merged**